lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/MultiRuntimeConfigurationBuilder.java

```diff
@@ -135,15 +137,15 @@ public Configuration build() {
       conf = conf.bindSetEntry(RuntimeNames.class, runtimeName);
     }
 
-    conf = conf.bindImplementation(
-            MultiRuntimeDefinitionGenerator.class, MultiRuntimeDefinitionGeneratorImpl.class);
-
-    if(!this.submissionRunitme.equalsIgnoreCase(RuntimeIdentifier.RUNTIME_NAME)){
-      throw new RuntimeException("Unsupported submission runtime " + this.submissionRunitme);
+    if(!this.submissionRuntime.equalsIgnoreCase(RuntimeIdentifier.RUNTIME_NAME)){
+      throw new RuntimeException("Unsupported submission runtime " + this.submissionRuntime);
     }
 
+    conf.bindImplementation(MultiRuntimeBootstrapConfigurationGenerator.class,
```

Comment:
From looking on the implementation this call actually mutates the state, so assignment is optional.